devtools/clean_workspace.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
In the other scripts, I use the term "workspace" for the workspace defined in the root Cargo.toml, i.e. the packages/*. What about
- not cleaning contracts here
- rename to just `clean.sh`

Reply:
Sure.